src/routes/grantform/GrantForm.js

```diff
@@ -0,0 +1,145 @@
+      totalCost = '',
+      description = '',
+      summary = '',
+    } = this.props;
+
+    return (
+      <div className={s.container}>
+        <div className={s.content}>
+          <div className={s.labelContainer}>
+            <div className={s.label}>Contact</div>
+          </div>
+
+          <input
+            type="text"
+            value={name}
+            placeholder="Name"
+            className={s.input}
+            onChange={({ target: { value } }) => inputChange('name', value)}
+          />
+
+          <div className={s.contactContainer}>
+            <input
+              type="text"
+              value={email}
+              placeholder="Email"
+              className={`${s.input} ${s.email}`}
+              onChange={({ target: { value } }) => inputChange('email', value)}
+            />
+
+            <input
+              type="text"
+              value={phone}
+              placeholder="Phone number"
+              className={`${s.input} ${s.phone}`}
+              onChange={({ target: { value } }) => inputChange('phone', value)}
+            />
+          </div>
+
+          <div className={s.labelContainer}>
+            <div className={s.label}>About your event</div>
+          </div>
+
+          <DatePicker
+            selected={date}
+            placeholderText="Date"
+            dateFormat="DD/MM/YYYY"
+            onChange={(value) => inputChange('date', value)}
+            locale="en"
+            className={`${s.input} ${s.date}`}
+          />
+
+          <input
+            type="text"
+            value={summary}
+            placeholder="Short summary"
+            className={s.input}
+            onChange={({ target: { value } }) => inputChange('summary', value)}
+          />
+
+          <textarea
+            type="text"
+            value={description}
+            placeholder="Description"
+            className={`${s.input} ${s.description}`}
+            onChange={({ target: { value } }) => inputChange('description', value)}
+          />
+
+          <div className={s.labelContainer} >
+            <div className={s.label}>Grant</div>
+          </div>
+
+          <div className={s.costContainer}>
+            <input
+              type="text"
+              value={askAmount}
+              placeholder="How much money do you need?"
+              className={`${s.input} ${s.ask}`}
+              onChange={({ target: { value } }) => inputChange('askAmount', value)}
+            />
+
+            <input
+              type="text"
+              value={totalCost}
+              placeholder="How much does the event cost?"
+              className={`${s.input} ${s.totalCost}`}
+              onChange={({ target: { value } }) => inputChange('totalCost', value)}
+            />
+          </div>
+
+          <div className={s.captchaContainer}>
+            <ReCAPTCHA
+              // ref="recaptcha"
+              sitekey="6LdRlycTAAAAAHFDPQDPkspMIqEDnUR4xk7ouueV"
```

Comment:
Yeah, that makes sense. It is a public key but might be used in more places in the future.